scripts/preload-images.sh

```diff
@@ -17,7 +17,7 @@ set -e
 
 # Create array for images to preload
 IMAGES_TO_PRELOAD=(
-	httpd:2.4.58
+	docker.io/library/httpd:2.4.58
```

Comment:
Does this need to be specified?  I think it defaults to dockerhub?

Reply:
when using podman, it actually asks, which pauses the workflow, so I expanded it